server/images.go

```diff
@@ -1178,7 +1178,7 @@ var deployment = sync.OnceValue(func() string {
 		for scanner.Scan() {
 			line := scanner.Text()
 			if anchorPoint.MatchString(line) {
-				return " container"
+				return "Container"
```

Comment:
While I know this won't detect other non-Docker instances it might be safer to refer to the actual "product" used here vs "Container". It's fine if the user agent is not set for other ones for now as we can address this in the future.

```
func containerProduct() string {
    if _, err := os.Stat("/.dockerenv"); err == nil {
        return "Docker"
    }
    return ""
}
```